src/util.ts

```diff
@@ -280,3 +297,36 @@ export const addHashToBaseline = (hash: string, filepath: string): void => {
     ignoreMessages: baselineErrorsFile.meta.ignoreMessages
   })
 }
+
+export const toGitLabOutputFormat = (
+  errorSummaryMap: ErrorSummaryMap,
+  specificErrorMap: SpecificErrorsMap,
+  errorOptions: ErrorOptions
+): string => {
+  const result: GitLabErrorFormat[] = []
+
+  for (const [key, error] of errorSummaryMap.entries()) {
+    const specificErrors: SpecificError[] = getSpecificErrorsMatchingSummary(
```

Comment:
You do not need to add `: SpecificError[]` here, since it is already the return type of `getSpecificErrorsMatchingSummary`, so the type is inferred.

But it does not hurt either.